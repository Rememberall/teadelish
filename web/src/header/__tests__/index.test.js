import React from 'react';
import renderer from 'react-test-renderer';

import Header from '../index';

it('should not change', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
