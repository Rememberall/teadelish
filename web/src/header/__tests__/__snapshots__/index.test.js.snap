// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should not change 1`] = `
<header>
  teadelish
</header>
`;
